docs/content/querying/lookups.md

```diff
@@ -3,42 +3,82 @@ layout: doc_page
 ---
 # Lookups
 
-Lookups are a concept in Druid where dimension values are (optionally) replaced with a new value. See [dimension specs](../querying/dimensionspecs.html) for more information. For the purpose of these documents, a "key" refers to a dimension value to match, and a "value" refers to its replacement. So if you wanted to rename `appid-12345` to `Super Mega Awesome App` then the key would be `appid-12345` and the value would be `Super Mega Awesome App`. 
+Lookups are a concept in Druid where dimension values are (optionally) replaced with a new value. 
+See [dimension specs](../querying/dimensionspecs.html) for more information. For the purpose of these documents, 
+a "key" refers to a dimension value to match, and a "value" refers to its replacement. 
+So if you wanted to rename `appid-12345` to `Super Mega Awesome App` then the key would be `appid-12345` and the value 
+would be `Super Mega Awesome App`. 
 
-It is worth noting that lookups support use cases where keys map to unique values (injective) as per a country code and a country name, and also supports use cases where multiple IDs map to the same value as per multiple app-ids belonging to a single account manager.
+It is worth noting that lookups support use cases where keys map to unique values (injective) as per a country code and 
```

Comment:
"as per" means "in accordance with", and I don't think that is what is meant here.  This is an example, yes?  So this could say "e.g., a country code and a country name".  
(And same on the next line.)